quantum/rgblight/rgblight.h

```diff
@@ -246,17 +246,19 @@ extern bool           is_rgblight_initialized;
 
 // Should stay in sycn with rgb matrix config as we reuse eeprom storage for both (for now)
```

Comment:
```suggestion
```
This comment is wrong — the EEPROM storage for rgblight and RGB Matrix was separated back in #5965.

Also the `speed` member and related things like `rgblight_increase_speed()` appeared here in #2922, because someone had a great idea to just reuse the rgblight configuration for RGB Matrix; they still don't do anything for rgblight, except the changes in this PR make the `speed` value persistent.